### ISSUE TITLE
Crashing when attempting to craft Void Cells

### ISSUE THREAD

**Issue by Reignofmagic** [ReikaKalseki/Reika_Mods_Issues#221]
This is similar to both of my previous issues #201 and #209 

Note Since then I have updated all reika mods to the latest currently available version (8c/a where applicable)

Below is the pastbin of the initial crash when I completed the kuro repeater network. (I generally have to fly the entire line of repeaters to get them to work, although after the first crash this has not been the case)
I'll edit this issue with more crashlogs if I encounter any more crashes. 

Crash 1: http://pastebin.com/gDexsj7X
--- Note when I logged in next the first void cell was completed (had materials to make 8).

Edit- Crash 2: http://pastebin.com/gqy54BKV
--- Happened after I had crafted an additional 4 void cells, it also completed the cell upon next log in.

**Comment by ReikaKalseki** [comment]
Clearly something is wrong with your instance. It seems this is crashing every time you attempt to draw from the crystal network (in fact, since I put some try-catch in v8, it seems any access to it at all triggers CMEs). Remove every other mod from the instance and try again, because something is either rewriting or multithreading something it should not.

**Comment by ljfa-ag** [comment]
The only thing I could imagine being problematic would be Optifine...did you try it without?

**Comment by ReikaKalseki** [comment]
Optifine is rendering. This is world access.

**Comment by Reignofmagic** [comment]
@ReikaKalseki 
@ljfa-ag 
I do have optifine set to multi-thread chunk loading if that'd be an issue. As for the crashing every time. It seems to crash every time on certain things (previously the item fabricrator). Others have no problems, and a few have random problems (these void cells). I'll be preforming all tests with the fabricator since that was causing me problems every time.

Here are my tests to see what the deal is, done on version 8c for the sake of consistency with previous crashes. 

Pre Steps: make 2 backups of current world

Attempt 1.

Step 1: delete crystalnet.dat file from world folder in copy 2 (my crystal network doesn't seem to save every time. For example kuro never will load up until I fly the entire route of the repeaters)
Step 2, use old mod file
step 3: try once new dat file has been generated & crystal network is fully working
Result: Crash ( http://pastebin.com/Q4xx9fPM )

Attempt 2.

Step 1. Make new mods folder
Step 2. put only reika mods in with nei
Step 2.5: cry because I have to re-level up the casting altar for some reason
Step 3. test
Result: No crash, but I couldn't get kuro to go to the altar. even tho it was connected. It would just sit there attempting to craft. I let it sit for 10 minutes. 

Attempt 3. Repeat attempt 2, but add chicken chunks to chunkload every kuro pylon
again kuro would not be used with the altar, but it would stream to a item fabricator

-- Update
Im stupid it was the dark gray, not black... it works just fine. I'll start adding mods until it will no longer craft

--Update 2: 
After adding in some mods my world freeze upon world load. Is there any way to easily level up the altar and make a crystal network for test world purposes?

**Comment by ReikaKalseki** [comment]
I have Optifine with no issues, but try removing it and seeing if that fixes the issue.

There is no machine sensitivity in this; they all call the same code.

**Comment by Reignofmagic** [comment]
@ReikaKalseki 

Well I tried removing quite a few mods to see if I could narrow down my search. Over the course of a few hours I have determined that at least a sigificant portion of the problem stems from draconic evolution (1.0.1c). I was able to create around 8 item fabricators before I crashed when I had it removed. If I added it back I'd crash promptly on the start of the ritual. I'll test with all the other mods added back in tomorrow, and then try with an updated draconic if available. 

Not really sure why it would be causing problems with your mods, but my guess is my install is just messed up slightly somehow. 

Also as an aside I installed just your mods in a completely different directory (a new multi-mc instance), freshly downloaded, with only a few other mods (nei, and a few of its addons, thaumcraft, and ender IO with a few others), and I was still having the odd problem where I can't see any crafting recipes in the lexicon that I told you about back a month or two ago on the forums. So I'm at a loss as to why that is happening. it isn't a problem as NEI works for getting the recipes no problem. Just slightly odd in the uniqueness of it.  My guess is the crafting altar problem also stems, or is at least somewhat connected from this odd issue I'm having as well. 

Anyway I wouldn't worry about this too much as it is isolated to just me. I'll get back to you with the results of my tests sometime tomorrow involving draconic evolution.

**Comment by ReikaKalseki** [comment]
> (a new multi-mc instance), freshly downloaded, with only a few other mods (nei, and a few of its addons, thaumcraft, and ender IO with a few others), and I was still having the odd problem where I can't see any crafting recipes in the lexicon that I told you about back a month or two ago on the forums

If it is a new instance (and thus world), you do not have the progression to see any recipes...

**Comment by Reignofmagic** [comment]
> If it is a new instance (and thus world), you do not have the progression to see any recipes...

@ReikaKalseki 

I understand that, it was for the tutorials I was making prior to V8 being released (I'll be re-recording them in-case anything changed aside from the crystal bee progression you removed). I had progressed up to the 2nd tier of casting temple, but still did not have the recipes showing in the books. 

Regardless since these issues seem to be isolated to just me I wouldn't worry about it. As for the tests results, I did a couple earlier before I went in to work I added about half of the mods I removed, excluding draconic since I thought it was causing the crashes, but it will only happen when multiple mods are back in. Each mod put back in by itself doesn't cause the crash to re-appear, at least with consistency. 

Anyway sorry for the confusion/distractions I hope you have a good rest of the day.
